src/QirRuntime/public/QirRuntime.hpp

```diff
@@ -160,7 +162,6 @@ extern "C"
     // Initializes the callable with the provided function table and capture tuple. The capture tuple pointer
     // should be null if there is no capture.
     typedef void (*t_CallableEntry)(PTuple, PTuple, PTuple);                                                // NOLINT
-    typedef void (*t_CaptureCallback)(PTuple, int64_t);                                                     // NOLINT
     QIR_SHARED_API QirCallable* quantum__rt__callable_create(t_CallableEntry*, t_CaptureCallback*, PTuple); // NOLINT
```

Comment:
This `typedef void (*t_CaptureCallback)(PTuple, int64_t)` was a duplicate. Removed.